<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Peter Humburg" />
  <title>Rare variants</title>
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
  <link rel="stylesheet" href="/css/reveal.min.css"/>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="/css/theme/bug.css" id="theme">
    <!-- If the query includes 'print-pdf', include the PDF print sheet -->
    <script>
      if( window.location.search.match( /print-pdf/gi ) ) {
        var link = document.createElement( 'link' );
        link.rel = 'stylesheet';
        link.type = 'text/css';
        link.href = '/css/print/pdf.css';
        document.getElementsByTagName( 'head' )[0].appendChild( link );
      }
    </script>
    <!--[if lt IE 9]>
    <script src="/lib/js/html5shiv.js"></script>
    <![endif]-->
    <script src="/MathJax/MathJax.js?config=TeX-AMS-MML_HTMLorMML,local/local" type="text/javascript"></script>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">Rare variants</h1>
  <h1 class="subtitle">Discovery and interpretation</h1>
    <h2 class="author">Peter Humburg</h2>
    <h3 class="date">18<sup>th</sup> November 2015</h3>
</section>

<section><section id="introduction" class="titleslide slide level1"><h1>Introduction</h1></section><section id="interpreting-patient-genomes" class="slide level2">
<h1>Interpreting patient genomes</h1>
<ul>
<li>Sequencing of patient genomes increasingly common</li>
<li>Can identify relevant variants</li>
<li>… amongst a large number of unrelated variants</li>
<li>… can be difficult to interpret</li>
<li><span class="fragment highlight-current-red"> Computational strategies critical to obtaining good set of candidates</span></li>
</ul>
<aside class="notes">
<p>Typical sequencing studies either focus on individual patients (or trios) or larger cohorts.</p>
<p>Obviously these serve different objectives, either individual diagnosis/treatment or generally gaining better understanding of phenotype.</p>
<p>Variants of interest may be rare or common. Will focus on rare variants in this talk. Some of this also applies to cancer genomes but won’t discuss that here.</p>
</aside>
</section><section id="finding-rare-disease-related-variants" class="slide level2">
<h1>Finding rare disease related variants</h1>
<p>Rare variants are of particular interest</p>
<ul>
<li>May have large effects</li>
<li><p>… but can be hard to find.</p></li>
<li>Need large sample sizes</li>
<li><p>… but may still struggle to identify causal variants.</p></li>
</ul>
<aside class="notes">
<p>No guarantee that there are rare variants with moderate/large effects in a given disease.</p>
<p>Even if they exist they likely to be only one of several factors involved in disease risk and may be rare even in the disease population.</p>
</aside>
</section></section>
<section><section id="identifying-novel-breast-cancer-risk-variants" class="titleslide slide level1"><h1>Identifying Novel Breast Cancer Risk Variants</h1></section><section id="motivation" class="slide level2">
<h1>Motivation</h1>
<ul>
<li>Several DNA repair genes implicated in breast and ovarian cancer susceptibility.</li>
<li>Strong evidence that rare loss-of-function variants confer increased risk.</li>
<li>Sequencing large number of patients not carrying known risk variants should lead to discovery of new ones.</li>
</ul>
</section><section id="study-design" class="slide level2">
<h1>Study design</h1>
<div class="twocolumn">
<ul>
<li>Exons of 507 DNA repair genes in 1,150 unrelated patients.</li>
<li>Pools of 24 individuals.</li>
<li>Included 79 individuals with known mutations in breast cancer predisposition genes as positive controls.</li>
</ul>
<div class="fragment">
<ul>
<li>No controls.</li>
<li>No barcoding.</li>
<li>Expect to do lots of Sanger sequencing in follow-up.</li>
</ul>
</div>
</div>
<aside class="notes">
<p>Note that no controls were sequenced and samples are pooled (no barcoding) to reduce amount of time and money required.</p>
<p>69 individuals also had ovarian cancer.</p>
</aside>
</section><section id="analysis-strategy" class="slide level2">
<h1>Analysis strategy</h1>
<ul>
<li>Sequence pools with <span class="fragment fade-out" data-fragment-index="1">GAIIx</span> <span class="fragment fade-in" data-fragment-index="1">HiSeq2000.</span></li>
<li>Call variants in pools with <a href="http://sourceforge.net/projects/syzygy/">Syzygy</a>.</li>
<li>Annotate variants to identify loss of function.</li>
<li>Validate variants of interest.</li>
<li>Sequence relevant genes in control panel.</li>
</ul>
<aside class="notes">
<ul>
<li>Aim for <span class="math">\(\gt\)</span> 10<span class="math">\(\times\)</span> coverage per genome in target regions.</li>
<li>Should be able to detect singletons in pool.</li>
<li>But of course coverage isn’t uniform.</li>
</ul>
</aside>
</section><section id="achieved-coverage" class="slide level2 small">
<h1>Achieved coverage</h1>
<figure>
<img src="figure/coverage.png" />
</figure>
<p><span class="math">\(\gt\)</span> 480<span class="math">\(\times\)</span> coverage in 90% of target region</p>
<aside class="notes">
<ul>
<li>filtered reads to exclude
<ul>
<li>ambiguous alignments (MQ == 0)</li>
<li>masked bases with quality &lt; 22</li>
</ul></li>
<li>poor performance for pool 22</li>
</ul>
</aside>
</section><section id="variant-calling" class="slide level2">
<h1>Variant calling</h1>
<ul>
<li>Syzygy called 34,564 variants in target region.</li>
</ul>
<div class="fragment">

<ul>
<li>Performance for known variants:
<ul>
<li>439/439 common SNPs</li>
<li>24/26 rare SNPs</li>
<li>51/54 rare (short) indels
<div>
</li>
</ul></li>
</ul>
<aside class="notes">
<ul>
<li>Sensitivity for rare SNPS: 92%</li>
<li>Sensitivity for rare indels: 94%</li>
</ul>
</aside>
</section></section>
<section><section id="aside-annotating-variants" class="titleslide slide level1"><h1>Aside: Annotating variants</h1></section><section id="a-simple-plan" class="slide level2">
<h1>A simple plan</h1>
<ul>
<li>Use EnsEMBL annotations (via Perl API)</li>
<li>Identify protein truncating variants</li>
<li>Group variants by gene to identify candidates for follow-up</li>
</ul>
<div class="fragment">
<p>But it isn’t that easy…</p>
<aside class="notes">
<ul>
<li>VEP was in its infancy at the time, using Perl API allowed for much more flexibility.</li>
<li>In hindsight probably not the best choice. API not stable, hard to maintain code based on it. Also pretty slow.</li>
</ul>
</aside>
</div>
</section><section id="beware-of-transcript-annotations" class="slide level2">
<h1>Beware of transcript annotations</h1>
<p><a href="http://grch37.ensembl.org/Homo_sapiens/Location/View?r=11:8149771-8149831;db=core", target="_blank"> <img src="figure/transcripts.png" /> </a></p>
<div class="footnote">
<p><a href="http://www.genomemedicine.com/content/6/3/26" target="_blank"> McCarthy <em>et al.</em> Genome Medicine 2014 6:26 </a></p>
</div>
<aside class="notes">
<ul>
<li>EnsEMBL contains large number of transcripts.</li>
<li>Not all transcripts well supported by evidence.</li>
<li>Reporting the most severe consequence will enrich for false positives.</li>
<li>Including transcripts that aren’t expressed massively increases false positive annotations for PTV.</li>
</ul>
</aside>
</section><section id="beware-of-edge-effects" class="slide level2">
<h1>Beware of edge effects</h1>
<p><a href="http://grch37.ensembl.org/Homo_sapiens/Variation/Explore?db=core;r=6:30557978-30558977;v=rs72545970;vdb=variation;vf=116290482", target="_blank"> <img src="figure/indel.png" /> </a></p>
<div class="footnote">
<p><a href="http://www.genomemedicine.com/content/6/3/26" target="_blank"> McCarthy <em>et al.</em> Genome Medicine 2014 6:26 </a></p>
</div>
<aside class="notes">
<ul>
<li>At the time of the study EnsEMBL incorrectly annotated this as <em>stop loss</em>.</li>
<li>More recent versions of VEP correctly note that this is <em>frame shift/stop retained</em>.</li>
<li>But note that the most severe consequence still is <em>frame shift</em>.</li>
</ul>
</aside>
</section><section id="beware-of-misaligned-indels" class="slide level2">
<h1>Beware of misaligned indels</h1>
<p><a href="http://www.genomemedicine.com/content/7/1/76/figure/F1", target="_blank"> <img src="figure/misaligned_indel.jpg" /> </a></p>
<div class="footnote">
<p><a href="http://www.genomemedicine.com/content/7/1/76" target="_blank">Münz <em>et al.</em> Genome Medicine 2015 7:76</a></p>
</div>
<aside class="notes">
<ul>
<li>Indel position may be ambiguous.</li>
<li>Variant callers typically report the <em>left most</em> position, i.e. position closest to 5’ end of forward strand.</li>
<li>Really should report position closest to 3’ end of transcript.</li>
</ul>
</aside>
</section></section>
<section><section id="back-to-the-breast-cancer-study" class="titleslide slide level1 unnumbered"><h1>Back to the Breast Cancer Study</h1></section><section id="selecting-candidate-genes" class="slide level2">
<h1>Selecting candidate genes</h1>
<div class="twocolumn">
<ul>
<li>Identified 1,044 PTVs</li>
<li>Ranked genes by number of truncating mutations observed.</li>
<li>Identify candidate genes</li>
</ul>
<div class="fragment">
<ul>
<li>Top ranking genes were BRCA2, CHEK1, ATM, BRCA1, …</li>
<li>Partially driven by positive controls.</li>
<li>First interesting gene on list was PPM1D with 5 PTV.</li>
<li>None of these PTVs present in 1000 Genomes.</li>
</ul>
</div>
</div>
</section><section id="investigating-ppm1d" class="slide level2" data-transition="none">
<h1>Investigating PPM1D</h1>
<figure>
<img src="figure/ppm1d_2.png" />
</figure>
<ul>
<li>PPM1D is a phosphatase</li>
<li>Phosphatase domain encoded by first 5 exons</li>
</ul>
</section><section id="investigating-ppm1d-1" class="slide level2" data-transition="none">
<h1>Investigating PPM1D</h1>
<figure>
<img src="figure/ppm1d_3.png" />
</figure>
<p>All identified truncating mutations validated with Sanger sequencing.</p>
</section><section id="phase-2-case-control-study" class="slide level2" data-transition="none">
<h1>Phase 2: Case-control study</h1>
<figure>
<img src="figure/ppm1d_4.png" />
</figure>
<ul>
<li>Sequenced PPM1D an additional 2456 cases and 1347 controls.</li>
<li>Identified 10 additional PTVs (none in controls)</li>
</ul>
<aside class="notes">
<ul>
<li>All PTVs clustering in last exon</li>
<li>proceeded to only sequence this region.</li>
</ul>
</aside>
</section><section id="phase-2-case-control-study-1" class="slide level2" data-transition="none">
<h1>Phase 2: Case-control study</h1>
<figure>
<img src="figure/ppm1d_5.png" />
</figure>
<ul>
<li>Sequenced final exon only in 5325 cases and 4514 controls.</li>
<li>Identified 15 additional PTVs in cases (1 in controls)</li>
</ul>
</section><section id="case-control-summary" class="slide level2">
<h1>Case-control summary</h1>
<table>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: center;">Breast cancer</th>
<th style="text-align: center;">Ovarian cancer</th>
<th style="text-align: center;">controls</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Sequenced</td>
<td style="text-align: center;">6,912</td>
<td style="text-align: center;">1,121</td>
<td style="text-align: center;">5,861</td>
</tr>
<tr class="even">
<td style="text-align: left;">with PTV</td>
<td style="text-align: center;">18</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative risk</td>
<td style="text-align: center;">2.7</td>
<td style="text-align: center;">11.5</td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">95% CI</td>
<td style="text-align: center;">1.3 - 5.3</td>
<td style="text-align: center;">4.3 - 30.4</td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
</section><section id="how-does-it-work" class="slide level2">
<h1>How does it work?</h1>
<figure>
<img src="figure/pathway.png" />
</figure>
<div class="fragment">
<p>Cells expressing truncated versions of PPM1D show reduced activation of p53 in response to ionizing radiation.</p>
<aside class="notes">
<p>Suggests truncated PTV is hyperactive.</p>
</aside>
</div>
</section></section>
<section><section id="the-plot-thickens" class="titleslide slide level1 unnumbered"><h1>The Plot Thickens</h1></section><section id="a-complication" class="slide level2">
<h1>A complication</h1>
<div class="twocolumn">
<ul>
<li>Read counts for variant alleles appear low.</li>
<li>Difficult to assess in pools but also visible in trace data.</li>
<li>Consistently low frequency of PTVs.</li>
</ul>
<div class="fragment">
<figure>
<img src="figure/trace.png" />
</figure>
</div>
</div>
</section><section id="somatic-variation" class="slide level2">
<h1>Somatic variation?</h1>
<div class="twocolumn">
<ul>
<li>Could indicate that these are somatic mutations</li>
<li>If these are germ line variants we should see them in children of carriers.</li>
</ul>
<div class="fragment" data-fragment-index="1">
<figure>
<img src="figure/family.png" />
</figure>
</div>
</div>
<aside class="notes">
<ul>
<li>No guarantee that there are informative families.</li>
<li>Got lucky and found a few.</li>
</ul>
</aside>
</section><section id="further-complications" class="slide level2">
<h1>Further complications</h1>
<div class="twocolumn">
<ul>
<li>PPM1D truncating mutations appear to be mosaic in lymphocytes.</li>
<li>What does PPM1D look like in the tumour tissue?
<ul>
<li>Deep sequencing of DNA from tumour, stromal tissue and blood in four cases.</li>
<li>Found expected mutations in blood <span class="fragment highlight-red" data-fragment-index="1"><span class="fragment" data-fragment-index="1">but not in tumour or stroma</span></span></li>
</ul></li>
</ul>
<div class="fragment" data-fragment-index="1">
<figure>
<img src="figure/tumour_seq.png" />
</figure>
</div>
</div>
</section></section>
<section><section id="discussion" class="titleslide slide level1"><h1>Discussion</h1></section><section id="possible-interpretations" class="slide level2">
<h1>Possible interpretations</h1>
<ul>
<li>Are these mutations present in cell of cancer origin but lost later?</li>
<li>Is oncogenesis driven by lymphocytes?</li>
<li>Are the PPM1D mutations only symptoms of an underlying problem that leads to cancer development in other tissues?</li>
<li>Are PPM1D mutations and cancer unrelated?</li>
</ul>
</section><section id="loss-during-cancer-development" class="slide level2">
<h1>Loss during cancer development</h1>
<ul>
<li>Evidence for loss of heterozygosity at PPMID locus.</li>
<li>The lost haplotype is the one carrying the PTV in lymphocytes.</li>
<li>Unclear whether the mutation was present prior to LOH event.</li>
<li>Loss of heterozygosity in this region is common in breast and ovarian cancers.</li>
</ul>
<aside class="notes">
<ul>
<li>Lends some support to the hypothesis that mutations were initially present</li>
<li>LOH may well be unrelated</li>
<li>Evidence is inconclusive</li>
</ul>
</aside>
</section><section id="oncogenesis-driven-by-lymphocytes" class="slide level2 small">
<h1>Oncogenesis driven by lymphocytes</h1>
<ul>
<li>Only real evidence is absence of mutation in tumour.</li>
<li>Unclear what the mechanism would be.</li>
</ul>
</section><section id="symptom-of-a-bigger-problem" class="slide level2">
<h1>Symptom of a bigger problem</h1>
<ul>
<li>Could be a sign of general genome instability.</li>
<li>This might lead to clonal expansion of cells with PPM1D PTVs as well as cancers.</li>
<li>Unclear what the driver of this would be.</li>
</ul>
</section><section id="simply-unrelated" class="slide level2">
<h1>Simply unrelated</h1>
<ul>
<li>Evidence supporting some relationship between PPM1D PTVs and cancer seems strong.</li>
<li>Observation has been replicated by <a href="http://jnci.oxfordjournals.org/content/early/2013/11/18/jnci.djt323.full">Akbari <em>et al.</em>, 2013</a></li>
<li>Somatic PPM1D PTVs have been found in cancers (<a href="http://jcb.rupress.org/content/201/4/511.full">Kleiblova <em>et al.</em>, 2013</a>, <a href="http://www.nature.com/ng/journal/v46/n7/full/ng.2995.html">Zhang <em>et al.</em>, 2014</a>)</li>
</ul>
</section></section>
<section><section id="lessons-learned" class="titleslide slide level1"><h1>Lessons Learned</h1></section><section id="finding-rare-variants" class="slide level2">
<h1>Finding rare variants</h1>
<ul>
<li>Strategy to sequence as many cases as possible paid off.</li>
<li>Would not have found PPM1D PTVs if we had split initial sequencing between cases and controls.</li>
<li>A lot, but very focused, follow-up required.</li>
<li>Focus on candidate gene panel paid off for similar reasons</li>
<li>… but means we have no easy way to check for other shared genomic variation amongst PPM1D PTV carriers.</li>
</ul>
</section><section id="somatic-variation-1" class="slide level2">
<h1>Somatic variation</h1>
<div class="twocolumn">
<ul>
<li>Were lucky that study design was suited to discovery of somatic variation.</li>
<li>Can find somatic variants through deep sequencing</li>
<li>but proving that a variant is somatic can be difficult in absence of control tissue.</li>
<li>Variant frequency in gDNA and RNA may differ markedly.</li>
</ul>
<figure>
<img src="figure/gdna_cdna.png" />
</figure>
</div>
</section><section id="variant-annotation" class="slide level2" data-transition="none">
<h1>Variant annotation</h1>
<ul>
<li>Be careful with automated annotations.</li>
<li>Have improved a lot over the last few years</li>
<li>… but can still be misleading or incomplete.</li>
<li>Consider PPM1D PTVs</li>
</ul>
</section><section id="variant-annotation-1" class="slide level2" data-transition="none">
<h1>Variant annotation</h1>
<ul>
<li>Be careful with automated annotations.</li>
<li>Have improved a lot over the last few years</li>
<li>… but can still be misleading or incomplete.</li>
<li>Consider PPM1D PTVs
<ul>
<li>Truncation of final exon.</li>
<li>(Correctly) predicted to escape nonsense mediated decay.</li>
<li>So not loss of function.</li>
<li>Doesn’t mean we should ignore it!</li>
</ul></li>
</ul>
</section></section>
<section><section id="acknowledgements" class="titleslide slide level1" data-transition="none"><h1>Acknowledgements</h1></section><section id="acknowledgements-1" class="slide level2" data-transition="none">
<h1>Acknowledgements</h1>
<div class="twocolumn">
<h3 id="wtchg">WTCHG</h3>
<p><strong>Peter Donnelly</strong></p>
<p>Manuel Rivas</p>
<p>Andrew Rimmer</p>
<p>Davis McCarthy</p>
<h3 id="icr">ICR</h3>
<p><strong>Nazneen Rahman</strong></p>
<p>Elise Ruark</p>
<p>Katie Snape</p>
</div>
</section></section>
    </div>
  </div>


  <script src="/lib/js/head.min.js"></script>
  <script src="/js/reveal.min.js"></script>

  <script>

      // Full list of configuration options available here:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        controls: true,
        progress: true,
        history: true,
        center: true,
        slideNumber: true,
        theme: 'bug', // available themes are in /css/theme
        transition: Reveal.getQueryHash().transition || 'default', // default/cube/page/concave/zoom/linear/fade/none

        // Optional libraries used to extend on reveal.js
        dependencies: [
          { src: '/lib/js/classList.js', condition: function() { return !document.body.classList; } },
          { src: '/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
          { src: '/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
//          { src: '/plugin/search/search.js', async: true, condition: function() { return !!document.body.classList; } },
//          { src: '/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } },
          { src: '/plugin/math/math.js', async: true, condition: function() { return !!document.body.classList; }}
]});
//  Reveal.configure({slideNumber:true});
    </script>
    </body>
</html>